### DISCUSSION TITLE
Why debugging was made difficult from 1.15 to 1.16?

### DISCUSSION THREAD

**Discussion by neoceph** [microsoft/vscode-cmake-tools#3986] [Q&A] [answered]
I was extensively using vscode with CMake for one of my projects. In the past (version <=1.15), it was much easier to pick between CMake: Debug/Release, available compiler options, and target executable, including CTests. I simply needed to click around in the bottom panel (see snapshot)
![image](https://github.com/user-attachments/assets/347cdf38-cdcb-4f18-8699-5683b4f0ee8b)

But starting version 1.16 this has change significantly and was made difficult to switch around these options. How can I easily switch between compilation type?

![image](https://github.com/user-attachments/assets/350b4f9f-d36a-4cdb-9023-d5fc623009bf)

**Comment by gcampbell-msft** [comment, marked as answer]
@neoceph These options have moved to the project status view: 
![image](https://github.com/user-attachments/assets/68cd25b1-8b13-44d8-9a57-535f9c436fd7)

If desired, you can customize where some of these buttons appear by modifying these settings: `cmake.options.advanced`, `cmake.options.statusBarVisibility`
